WcaOnRails/app/controllers/api/v0/persons_controller.rb

```diff
@@ -14,6 +14,11 @@ def show
     render json: person_to_json(person)
   end
 
+  def results
+    person = Person.current.includes(:user, :ranksSingle, :ranksAverage).find_by_wca_id!(params[:wca_id])
```

Comment:
I guess you copied this from the methods above, but here we don't need the `.includes(:user, :ranksSingle, :ranksAverage)` part as we don't show any of these associations.